cmd/plural/cd.go

```diff
@@ -118,7 +118,17 @@ func (p *Plural) doInstallOperator(url, token, values string) error {
 	}
 
 	vals := map[string]interface{}{}
-	if values != "" {
+	if values == "" {
```

Comment:
We actually probably would want to merge the cli flag provided values and the global values if both are provided

Reply:
done